index.html

```diff
@@ -65,19 +62,22 @@ <h4>Contact</h4>
             </div>
         </nav>
         <div class="main-container">
+            <div class="nav-open-close">
+                <i class="fas fa-bars"></i>
+            </div>
             <div class="introduction">
                 <h2>
-                    Hi! I'm Prologue, a free responsive<br>
-                    site template designed by HTML5 UP.
+                    Hi! My name is Thorben Bender<br>
+                    I am a fullstack developer
                 </h2>
                 <p>
-                    Ligula scelerisque justo sem accumsan diam quis<br>
-                    vitae natoque dictum sollicitudin elementum.
+                    On this page, you can see some of the work i done
+                    and learn more about me.
                 </p>
-                <a href="">Magna Aliquam</a>
+                <a href="#about-me">About Me</a>
```

Comment:
I really like that you have decided to use your name and information, so you can use it as a portfolio site.